hazelcast-spring/src/main/resources/hazelcast-spring-5.3.xsd

```diff
@@ -6204,11 +6204,11 @@
         <xs:attribute name="factory-class-name" type="xs:string"/>
     </xs:complexType>
 
-    <xs:complexType name="alto-client">
+    <xs:complexType name="tpc-client">
         <xs:annotation>
             <xs:documentation>
-                Configures the client for Alto, which is the next generation Hazelcast that
-                uses thread-per-core model. Alto-aware clients will maintain connections to
+                Configures the client for tpc, which is the next generation Hazelcast that
```

Comment:
```suggestion
                Configures the client for TPC, which is the next generation Hazelcast that
```
should we use capitalized `TPC` for logs and documentation? If so, I can point to such places in my review

Reply:
Good idea. I just updated the PR.